test.py

```diff
@@ -9,18 +9,26 @@
 import tempfile
 import time
 
-
```

Comment:
Pep8 is a set of guifelines, not rules. We can and will have many deviations and toys is fine.